### ISSUE TITLE
Carousel announce unavailable on 'next' or 'previous' button click

### ISSUE THREAD

**Issue by Shemetiuk** [bootstrap-vue/bootstrap-vue#7117]
### Describe the bug
Carousel component didn't announce correctly when switching slides using 'next slide' or 'previous slide' buttons on Windows with NVDA. 

### Steps to reproduce the bug
You can use any carousel component which has 3 or more slides on page
https://bootstrap-vue.org/docs/components/carousel
On Windows (Chrome browser) with NVDA when you navigate to 'next slide' button and click on it - screen reader says 'Unavailable'.
On Mac (Chrome browser) with VoiceOver when you navigate to 'next slide' button and click on it - screen reader says nothing.

### Expected behavior
I'm not sure that screen reader should announce something but it definitely shouldn't announce `Unavailable`. 

### Versions
**Libraries:**
Maybe any but it's happening on
BootstrapVue: 2.16.0
and on your website which is using
BootstrapVue: 2.23.0
### Demo link
https://bootstrap-vue.org/docs/components/carousel
